File: Syncup-backend/models/Submission.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const submissionSchema = new Schema({
  evidenceUrl: {
    type: Schema.Types.String,
    required: true
  },
  completedBy: {
    type:mongoose.Schema.Types.ObjectId,
	required: true,
    ref:'User'
  },
  activity: {
    type:mongoose.Schema.Types.ObjectId,
    ref:'Activity',
    required: true
  },
});



const Submission = mongoose.model('Submission', submissionSchema);

module.exports = Submission;

File: Syncup-android/app/src/main/java/com/example/syncup_android/data/model/Submission.kt
package com.example.syncup_android.data.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class Submission(
    @JsonProperty("_id")
    val id: String,
    @JsonProperty("evidenceUrl")
    val evidenceUrl: String,
    @JsonProperty("completedBy")
    val completedBy: User,
    @JsonProperty("activity")
    val activity: Activity
)
